### DISCUSSION TITLE
why harbor only support postgresql db

### DISCUSSION THREAD

**Discussion by kfhanjie** [goharbor/harbor#17173] [Q&A]
why harbor only support postgresql db?   our company db team only support mysql ,  it's too risky and costly to runing postgresql by myself.

**Comment by chlins** [comment]
Harbor has already set up a multiple database workgroup from the community to work on this feature, and was planned initially to be delivered in the harbor v2.7. The proposal can be found here: https://github.com/goharbor/community/pull/188.
